src/dismantle/package/_handlers.py

```diff
@@ -282,7 +281,10 @@ def _fetch_and_extract(self):
         if req.status_code not in [200, 304]:
             raise FileNotFoundError(req.status_code)
         elif req.status_code == 200:
-            self._cache.parents[0].mkdir(parents=True, exist_ok=True)
+            # Make sure parent folder of the cache exists
+            self._cache.parents[0].mkdir(0o777, parents=True, exist_ok=True)
+            log.info(f'Creating dir {self._cache.parents[0]} to hold cache')
+            # Write the package raw bytes into a single cache file
             with open(self._cache, 'wb') as cached_package:
                 cached_package.write(req.content)
             self._updated = True
```

Comment:
Thanks! can you also bump the version to `1.1.0` in `src/dismantle/__version__.py`?